pkg/apis/events/v1beta1/register.go

```diff
@@ -41,5 +41,5 @@ func init() {
 	// We only register manually written functions here. The registration of the
 	// generated functions takes place in the generated files. The separation
 	// makes the code compile even when the generated files are missing.
-	localSchemeBuilder.Register(RegisterDefaults)
```

Comment:
should this have removed RegisterDefaults?

Reply:
Didn't know Default shouldn't be removed. I've added it back. Not sure if e2e test failure is related to this..